build.gradle

```diff
@@ -191,8 +191,9 @@ subprojects {
                 google_auth_credentials: 'com.google.auth:google-auth-library-credentials:0.4.0',
                 okhttp: 'com.squareup.okhttp:okhttp:2.5.0',
                 okio: 'com.squareup.okio:okio:1.6.0',
-                opencensus_api: 'io.opencensus:opencensus-api:0.7.0',
-                opencensus_impl: 'io.opencensus:opencensus-impl:0.7.0',
+                opencensus_api: 'io.opencensus:opencensus-api:0.8.0',
+                opencensus_contrib_grpc_metrics: 'io.opencensus:opencensus-contrib-grpc-metrics:0.8.0',
+                opencensus_impl: 'io.opencensus:opencensus-impl:0.8.0',
```

Comment:
Why do you depend on the implementation?

Reply:
It looks like it is only needed for the interop tests: https://github.com/grpc/grpc-java/blob/37a67bf04111c0a9d42a7722b89bab028c882000/interop-testing/build.gradle#L29